### ISSUE TITLE
The current machine does not support all of the following CPU features that are required by the image...

### ISSUE THREAD

**Issue by awesomecogs** [haberda/signal-addon#131]
After upgrading from 0.81.0 to 0.82.0 I was unable to send Signal messages from HA.

``` 
➜  ~ curl -X GET -H "Content-Type: application/json" 'http://192.168.1.10:8080/v1/about'
{"versions":["v1","v2"],"build":2,"mode":"native","version":"0.82","capabilities":{"v2/send":["quotes","mentions"]}}#

➜  ~ curl -X GET -H "Content-Type: application/json" 'http://192.168.1.10:8080/v1/groups/+1##########'
{"error":"The current machine does not support all of the following CPU features that are required by the image: [FP, ASIMD, AES, PMULL, SHA1, SHA2, CRC32, LSE, DCPOP, SHA3, SHA512, SVE, PACA, SVEBITPERM, SVE2].\nPlease rebuild the executable with an appropriate setting of the -march option."}#
``` 

### Home Assistant info:
**Platform**: Hardkernel ODROID-M1
**Core**: 2024.4.1
**Supervisor**: 2024.03.1
**Operating System**: 12.1
**Frontend**: 20240404.1

### Addon log
```
+ set -e
+ [ -z /config ]
+ usermod -u 1000 signal-api
usermod: no changes
+ groupmod -g 1000 signal-api
+ chown 1000:1000 -R /config
+ cat
+ cap_prefix=-cap_
+ cat /proc/sys/kernel/cap_last_cap
+ seq -s ,-cap_ 0 40
+ caps=-cap_0,-cap_1,-cap_2,-cap_3,-cap_4,-cap_5,-cap_6,-cap_7,-cap_8,-cap_9,-cap_10,-cap_11,-cap_12,-cap_13,-cap_14,-cap_15,-cap_16,-cap_17,-cap_18,-cap_19,-cap_20,-cap_21,-cap_22,-cap_23,-cap_24,-cap_25,-cap_26,-cap_27,-cap_28,-cap_29,-cap_30,-cap_31,-cap_32,-cap_33,-cap_34,-cap_35,-cap_36,-cap_37,-cap_38,-cap_39,-cap_40
+ [ native = json-rpc ]
+ hostname -I
+ awk {print $1}
+ export HOST_IP=172.30.33.2
+ exec setpriv --reuid=1000 --regid=1000 --init-groups --inh-caps=-cap_0,-cap_1,-cap_2,-cap_3,-cap_4,-cap_5,-cap_6,-cap_7,-cap_8,-cap_9,-cap_10,-cap_11,-cap_12,-cap_13,-cap_14,-cap_15,-cap_16,-cap_17,-cap_18,-cap_19,-cap_20,-cap_21,-cap_22,-cap_23,-cap_24,-cap_25,-cap_26,-cap_27,-cap_28,-cap_29,-cap_30,-cap_31,-cap_32,-cap_33,-cap_34,-cap_35,-cap_36,-cap_37,-cap_38,-cap_39,-cap_40 signal-cli-rest-api -signal-cli-config=/config
time="2024-04-06T16:41:44Z" level=info msg="Started Signal Messenger REST API"
[GIN] 2024/04/06 - 16:42:17 | 200 |     437.822µs |     10.255.0.26 | GET      "/v1/about"
[GIN] 2024/04/06 - 16:42:30 | 400 |    4.185708ms |     10.255.0.26 | GET      "/v1/groups/+1#########"
```

### /proc/cpuinfo
```
➜  ~ cat /proc/cpuinfo
processor       : 0
BogoMIPS        : 48.00
Features        : fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm lrcpc dcpop asimddp
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x2
CPU part        : 0xd05
CPU revision    : 0

processor       : 1
BogoMIPS        : 48.00
Features        : fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm lrcpc dcpop asimddp
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x2
CPU part        : 0xd05
CPU revision    : 0

processor       : 2
BogoMIPS        : 48.00
Features        : fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm lrcpc dcpop asimddp
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x2
CPU part        : 0xd05
CPU revision    : 0

processor       : 3
BogoMIPS        : 48.00
Features        : fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm lrcpc dcpop asimddp
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x2
CPU part        : 0xd05
CPU revision    : 0
```

Please let me know if there is any other information I can provide.

**Comment by awesomecogs** [comment]
I should've checked upstream... https://github.com/bbernhard/signal-cli-rest-api/issues/510.

**Comment by jellewie** [comment]
aww crap,  The application was throwing me only 400 errors after the latest update, 
`[GIN] 2024/04/07 - 11:15:49 | 400 |     133.332µs |     172.30.32.1 | GET      "/v1/receive/<phone>"`
so I decided to re-install it. but now I can not register anymore in "native mode".
![image](https://github.com/haberda/signal-addon/assets/1545223/be53510e-b83f-428b-b7c8-18e0b29b0ced)

I tried "json-rpc", but that also returned errors for me.
```
curl -X POST -H "Content-Type: application/json" --data '{"use_voice": false}' 'http://<ha IP>:8080/v1/register/<phone>'    
{"error":"Method requires valid account parameter"}#   
```
So I tried "Normal." Which works for now
```
[GIN] 2024/04/07 - 11:18:34 | 201 |  19.52342467s |     172.30.32.1 | POST     "/v2/send"
[GIN] 2024/04/07 - 11:23:22 | 200 | 12.965493089s |     172.30.32.1 | GET      "/v1/receive/+<phone>"
```

**Comment by haberda** [comment]
I think 0.83 should fix this issue.

**Comment by awesomecogs** [comment]
Yep, 0.83.0 fixed the issue.  Thanks!

On Mon, Apr 8, 2024, 8:15 AM Daniel Haber 'notifications at github.com'
<6v4dphot+reply+AKRAKJ45YIE6PYD2DGSVSBOED7HODEVBNHHIJYGSRI=

**Comment by jellewie** [comment]
Just to confirm, my issues have also been solved by 0.83. I'm back running on json-rpc
